app/test_main.py

```diff
@@ -2,2 +1,47 @@
+import pytest
 
-# write your code here
+from .main import get_human_age
+
+
+def test_add_not_int_argument() -> None:
+    with pytest.raises(TypeError):
+        get_human_age(12.52, "52.12")
+
+
+def test_negative_value() -> None:
+    assert (get_human_age(-12, -2) == [0, 0]), \
+        "The age shouldn't be negative"
```

Comment:
The test `test_negative_value` checks for negative input values, which is good. However, ensure that the `get_human_age` function is designed to handle negative values appropriately, as the current assertion expects `[0, 0]` as the result.